Summary Table/Summary Table Dev Flextable.Rmd

```diff
@@ -0,0 +1,398 @@
+---
+title: ''
+output: 
+  word_document:
+    reference_docx: "//conf/LIST_analytics/West Hub/02 - Scaled Up Work/RMarkdown/Locality Profiles/Summary Table/ST_ref_doc.docx"
+---
+;
+```{r setup, include = FALSE}
+knitr::opts_chunk$set(echo = FALSE, comment = NA)
+# Lines below for testing only:
+# LOCALITY <- "Falkirk West"
+# LOCALITY <- "Stirling City with the Eastern Villages Bridge of Allan and Dunblane"
+# LOCALITY <- "Mid-Argyll, Kintyre and Islay"
+# LOCALITY <- "City of Dunfermline"
+# LOCALITY <- "Barra"
+# LOCALITY <- "Whalsay and Skerries"
+# LOCALITY <- "Inverclyde East"
+ 
+# setwd("~/list-localities-profiles/")
+
+## Project file path
+ lp_path <- "/conf/LIST_analytics/West Hub/02 - Scaled Up Work/RMarkdown/Locality Profiles/"
+ 
+# Source in global functions/themes script
+ source("./Master RMarkdown Document & Render Code/Global Script.R")
+ 
+# # demographics
+ source("./Demographics/1. Demographics - Population.R")
+ source("./Demographics/2. Demographics - SIMD.R")
+#
+# # housing
+ source("./Households/Households Code.R")
+#
+# # general health
+ source("./General Health/3. General Health Outputs.R")
+#
+# # lifestyle & risk factors
+ source("./Lifestyle & Risk Factors/2. Lifestyle & Risk Factors Outputs.R")
+#
+# # unscheduled care
+ source("./Unscheduled Care/2. Unscheduled Care outputs.R")
+
+library(tidyverse)
+library(knitr)
+library(gridExtra)
+library(grid)
+library(kableExtra)
+library(dplyr)
+library(flextable)
+library(officer)
+library(officedown)
```

Comment:
Can these packages be added to `Master RMarkdown Document & Render Code/Global Script.R`? I think some of them are already loaded there so don't also need to be here either way :)